src/hotspot/share/gc/shared/collectedHeap.cpp

```diff
@@ -603,6 +622,37 @@ void CollectedHeap::post_initialize() {
   initialize_serviceability();
 }
 
+void CollectedHeap::log_gc_cpu_time() const {
+  LogTarget(Info, gc, cpu) out;
+  if (os::is_thread_cpu_time_supported() && out.is_enabled()) {
+    double process_cpu_time = os::elapsed_process_cpu_time();
+    double gc_cpu_time = elapsed_gc_cpu_time();
+    double string_dedup_cpu_time = UseStringDeduplication ? (double)os::thread_cpu_time((Thread*)StringDedup::_processor->_thread) / NANOSECS_PER_SEC : 0;
```

Comment:
If we consider stringdedup as gc-time, should this line be inlined to `elapsed_gc_cpu_time`?